tests/Common/ManagerConfigurationTestTrait.php

```diff
@@ -98,12 +99,12 @@ private function createFilledAssignmentsStorage(): AssignmentsStorageInterface
     {
         $storage = $this->createAssignmentsStorage();
 
-        $storage->add('Fast Metabolism', 'reader A');
-        $storage->add('reader', 'reader A');
-        $storage->add('author', 'author B');
-        $storage->add('deletePost', 'author B');
-        $storage->add('publishPost', 'author B');
-        $storage->add('admin', 'admin C');
+        $storage->add(new Assignment(userId: 'reader A', itemName: 'Fast Metabolism', createdAt: time()));
+        $storage->add(new Assignment(userId: 'reader A', itemName: 'reader', createdAt: time()));
+        $storage->add(new Assignment(userId: 'author B', itemName: 'author', createdAt: time()));
+        $storage->add(new Assignment(userId: 'author B', itemName: 'deletePost', createdAt: time()));
+        $storage->add(new Assignment(userId: 'author B', itemName: 'publishPost', createdAt: time()));
+        $storage->add(new Assignment(userId: 'admin C', itemName: 'admin', createdAt: time()));
```

Comment:
Seems, , `createdAt: time()` not need here...

Reply:
It's required argument in Assignment's constructor.